### PR TITLE
Custom about screen for Elite Backers

### PR DESCRIPTION
**Pull request by SerhiiGadzhilov**
[SYNERGY1-1591](https://symless.atlassian.net/browse/SYNERGY1-1591)

### CODE REVIEW

File: src/gui/src/MainWindow.cpp [hunk 1]
```diff
@@ -1266,8 +1267,19 @@ bool  MainWindow::on_m_pActionSave_triggered()
 
 void MainWindow::on_m_pActionAbout_triggered()
 {
+#ifndef SYNERGY_ENTERPRISE
+    if (appConfig().edition() == Edition::kBusiness) {
+        AboutDialog dlg(this, appConfig());
```
Comment (nbolton) [inline src/gui/src/MainWindow.cpp:1272]: How about... `AboutDialogBusiness`?

File: src/gui/src/MainWindow.cpp [hunk 2]
```diff
@@ -1266,8 +1267,19 @@ bool  MainWindow::on_m_pActionSave_triggered()
...
+        dlg.exec();
+    }
+    else {
+        AboutDialogEliteBackers dlg(this, appConfig());
```
Comment (nbolton) [inline src/gui/src/MainWindow.cpp:1276]: ...and `AboutDialogConsumer`?

**Review by nbolton — CHANGES_REQUESTED** [review]

File: src/gui/src/MainWindow.cpp (hunk 1 above)
Comment (SerhiiGadzhilov) [reply src/gui/src/MainWindow.cpp]: Fixed

File: src/gui/src/MainWindow.cpp (hunk 2 above)
Comment (SerhiiGadzhilov) [reply src/gui/src/MainWindow.cpp]: Fixed

**Review by nbolton — APPROVED** [review]

File: src/gui/src/MainWindow.cpp [hunk 3]
```diff
@@ -1266,8 +1267,19 @@ bool  MainWindow::on_m_pActionSave_triggered()
...
+    if (appConfig().edition() == Edition::kBusiness) {
+        AboutDialogBusiness dlg(this, appConfig());
+        dlg.exec();
+    }
+    else {
+        AboutDialogConsumer dlg(this, appConfig());
+        dlg.exec();
+    }
+#else
     AboutDialog dlg(this, appConfig());
```
[the remark this answers is not in the document]
Comment (SerhiiGadzhilov) [reply src/gui/src/MainWindow.cpp:1280]: Yes, I have tested this functionality with and without `-DSYNERGY_ENTERPRISE`.

**Pull request merged**